src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -469,6 +476,20 @@ internal async Task Eval(enum_EVALFLAGS dwFlags = 0)
                 }
                 else
                 {
+                    int numElements = 200;
+                    if (dwDAPFlags.HasFlag(DAPEvalFlags.CLIPBOARD_CONTEXT))
+                    {
+                        string showPrintElementsResult = await MIDebugCommandDispatcher.ExecuteCommand("show print elements", _debuggedProcess, ignoreFailures: true);
+                        // Possible values for 'numElementsStr'
+                        // "Limit on string chars or array elements to print is <number>."
+                        // "Limit on string chars or array elements to print is unlimited."
+                        string numElementsStr = Regex.Match(showPrintElementsResult, @"\d+").Value;
```

Comment:
>. [](start = 92, length = 1)

Don't ye need to check if the match was successful before you call '.Value'? Also, should you verify there is just one match?

Reply:
There is one capture context, so it should have only one Capture's value.